Documentation/gettingstarted/kata.rst

```diff
@@ -0,0 +1,106 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _kata:
+
+***************************
+Kata Containers with Cilium
+***************************
+
+`Kata Containers <https://katacontainers.io/>`_ is an open source project that
+provides a secure container runtime with lightweight virtual machines that feel
+and perform like containers, but provide stronger workload isolation using
+hardware virtualization technology as a second layer of defense.
+Similar to the OCI runtime ``runc`` provided by Docker, Cilium can be used with
+Kata Containers, providing a higher degree of security at the network layer and
```

Comment:
be used along with Kata Containers, using both enables higher degree of security. Kata Enhances